packages/flutter/lib/src/widgets/scroll_configuration.dart

```diff
@@ -73,15 +73,15 @@ class ScrollBehavior {
     switch (getPlatform(context)) {
       case TargetPlatform.iOS:
       case TargetPlatform.macOS:
-        return (PointerEvent ev) => IOSScrollViewFlingVelocityTracker();
+        return (PointerEvent ev) => IOSScrollViewFlingVelocityTracker(ev.kind);
```

Comment:
Can we rename `ev` to `event` here (and below) too?

Reply:
done